README.md

```diff
@@ -0,0 +1,9 @@
+# List of Software and Related Tools
+
+A wide variety of software tools are used by various stakeholders to 
+perform nuclear fuel cycle simulation, or interface with the results 
+of such simulation.  This list attempts to capture them based on 
+input from the community.
+
+* [Cyclus](https://fuelcycle.org) - an open-source, agent-based platform that uses a market mechanism to track flows of materials between facilities ([Github](https://github.com/cyclus/cyclus))
+    * [Cycamore](https://github.com/cyclus/cycamore) - an open-source collection of agents representing facilities, institutions and regions for use in Cyclus
```

Comment:
Minor thing: do we want these lines to be this long?

Also, if you want to include tools to interface with results, do we want to add cymetric?